try {
    throw "error"; // error: Uncaught exception: "error"
} finally {
    print("finally"); // expect: finally
}